### PR TITLE
add a base for testing with operator

### PR DESCRIPTION
**Pull request by mandydydy**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,9 +78,11 @@ default: load-balancer proxy target ipam nsp ctraffic frontend
 lint: 
 	golangci-lint run ./...
 
+NAMESPACE ?= red
+TESTWITHOPERATOR ?=false
```
Comment (LionelJouin) [inline Makefile:82]: Could you add _ between words

File: test/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+6. Configure Spire for the targets
+
+    ```bash
+    ./docs/demo/scripts/spire.sh meridio $ns
+    ```
+
+7. Install targets connected to trench-a
+
+    ```bash
+    helm install examples/target/helm/ --generate-name --create-namespace --namespace $ns --set applicationName=target-a --set default.trench.name=<trench-name> --set default.conduit.name=<conduit-name>
```
Comment (LionelJouin) [inline test/e2e/README.md:48]: <trench-name> could be set directly to "trench-a" and <conduit-name> we could precise to put load-balancer for helm and lb-fe for operator

File: test/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# E2E Test Instruction
```
Comment (LionelJouin) [inline test/e2e/README.md:1]: There is document related to e2e, might be good to remove it and add a link pointing to this one

File: test/e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+### Testing with Meridio Operator
+
+Meridio should test with the operator before each release to ensure the compatibility.
+
+9. Install the operator
+
+    ```bash
+    # run at the git root directory
+    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
+    cd Meridio-Operator
```
Comment (LionelJouin) [inline test/e2e/README.md:68]: command to pull git sub module and update it could be useful

File: test/e2e/manifest/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#! /bin/bash -x
```
Comment (LionelJouin) [inline test/e2e/manifest/deploy.sh:1]: Is this file used somewhere?

Comment (mandydydy) [reply test/e2e/manifest/deploy.sh:1]: This is just a helper script to deploy everything for operator testing. Maybe I should remove it

**Comment by mandydydy** [comment]
\reverify

**Review by LionelJouin — APPROVED** [review]

**Review by estinfra — COMMENTED** [review]
FOSS Introduction is done. Please check the [logs](https://artifactory.est.tech/artifactory/proj-est-nsm-generic-local/reports/meridio/master/1e2a700).

**Pull request merged**